src/auth.ts

```diff
@@ -187,9 +187,9 @@ export async function authorize(options: {
  * Loads the Apps Script API credentials for the CLI.
  * Required before every API call.
  */
-export async function loadAPICredentials(local = false): Promise<ClaspToken> {
```

Comment:
In some API calls, it makes sense to load only local credentials.
It would be nice to be explicit.

Reply:
Ok, then I think we need an `if/else` where if it is local only, then just check there. If it is false or does not specify, then do it recursively.